fuzz/fuzz_targets/mutate.rs

```diff
@@ -45,45 +45,56 @@ fuzz_target!(|bytes: &[u8]| {
     }
 
     // Mutate the Wasm with `wasm-mutate`. We always preserve semantics.
+    let mut wasm_mutate = wasm_mutate::WasmMutate::default();
+    wasm_mutate.seed(seed);
+    wasm_mutate.fuel(300);
+    wasm_mutate.preserve_semantics(true);
+
+    let mutated_wasm_iterator = wasm_mutate.run(&wasm);
+    let mut found = false;
+
+    match mutated_wasm_iterator {
+        Ok(mut iterator) => {
+            while let Some(mutated_wasm) = iterator.next() {
+                let features = WasmFeatures::default();
+                let mut validator = wasmparser::Validator::new();
+                validator.wasm_features(features);
+
+                match mutated_wasm {
+                    Ok(mutated_wasm) => {
+                        // Increase ony once for the same input Wasm
+                        if !found {
+                            // FIXME this metric is somehow undermeasuring the
+                            // effectiveness of the mutator now. Some modules
+                            // can have only one possible mutation meanwhile
+                            // others can have a lot and still each one of them
+                            // is counted as one if they have at least one
+                            // possible mutation
```

Comment:
This is working as intended: we are counting the number of modules we successfully mutate, not the number of possible mutations that could have been successful.